### ISSUE TITLE
Burmese Language Is Malformed

### ISSUE THREAD

**Issue by Elshara** [espeak-ng/espeak-ng#117]
The Burmese language needs to be reconfigured completely.

**Comment by rhdunn** [comment]
What do you mean by it being malformed?

What problems are you having with the language?

**Comment by Elshara** [comment]
Things seem to be spelled out by letter and numbers aren't reported.

**Comment by ValdisVitolins** [comment]
That's because for Burmese only [voice](https://github.com/espeak-ng/espeak-ng/blob/master/espeak-data/voices/sit/my) and [phonemes](https://github.com/espeak-ng/espeak-ng/blob/master/phsource/ph_burmese) are created, but it is missing rules files, how to spell text (`my_list` and `my_rules` files). Look at [language files](https://github.com/espeak-ng/espeak-ng/blob/master/docs/add_language.md#language-files).

**Comment by rhdunn** [comment]
https://github.com/espeak-ng/espeak-ng/tree/master/dictsource does contain `my_list` and `my_rules`.

The lack of number support is likely due to the commented out `//၁ 1`, etc. replace rules in `my_rules`. Words being spelled out letter-by-letter is most likely due to missing rules for certain Burmese letters.

To improve this, we need to know what words and numbers are being incorrectly handled/pronounced and feedback/input from a native speaker. @lwinmoe: Do you have any idea on what is causing this?

**Comment by ValdisVitolins** [comment]
Oh, sorry I somehow missed that there are `my_list` and `my_rules` files already, but they are too short.
Reece is right, to spell numbers correctly it is needed to add following lines in `my_list` file:
```
// numbers
_0      'abadaba // just example
_1      ...
_2      
_3      
_4      
_5      
_6      
_7      
_8      
_9      
_10     
_11     
_12     
_13     
_14     
_15     
_16     
_17     
_18     
_19     
_2X     
_3X     
_4X     
_5X     
_6X     
_7X     
_8X     
_9X     
_0C     
_1C     
_0M1    
_1M1    
_0M2    
_1M2    
_dpt    k'oma  // decimal comma
```
and uncomment lines in `my_rules` file:
```
၀ 0 // Convert numbers
၁ 1
၂ 2
၃ 3
၄ 4
၅ 5
၆ 6
၇ 7
၈ 8
၉ 9
```

**Comment by brawer** [comment]
For spelling out Burmese numbers, see [Unicode CLDR](https://www.unicode.org/repos/cldr/trunk/common/rbnf/my.xml) and the corresponding [demo](http://st.unicode.org/cldr-apps/numbers.jsp).

**Comment by valdisvi** [comment]
Yes, digits from 0 to 9 are spelled correctly, but written language is still garbled. I think [my_rules](https://github.com/espeak-ng/espeak-ng/blob/master/dictsource/my_rules) needs serious improvements.
